internal/pkg/cli/init.go

```diff
@@ -140,6 +142,13 @@ func (opts *InitOpts) loadApp() error {
 	return opts.initApp.Validate()
 }
 
+func (opts *InitOpts) loadEnv() {
+	if obj, ok := opts.initEnv.(*InitEnvOpts); ok {
+		// To deploy an application we need to specify its project.
+		obj.projectName = *opts.projectName
+	}
+}
```

Comment:
This function's existence feels odd to me. Would it be possible to remove it if we defer creation of the `InitEnvOpts` structure to when we have the project name or can we rename to something like `updateInitEnvProjectName`?

Reply:
Deleted it (see commented above)